Source/ReturnsExtensions.tt

```diff
@@ -58,5 +58,50 @@ namespace Moq
 <#
 		}
 		#>
+
```

Comment:
Could I ask you for a favour? While we're editing this file, could you please change the class definition in this file (line 15) from:

> ```csharp
> public static class GeneratedReturnsExtensions
> ```

to:

```csharp
static partial class ReturnsExtensions
```

and then, in `ReturnsExtensions.cs`:
* also declare `ReturnsExtensions` `partial`; and
* put an `[EditorBrowsable(EditorBrowsableState.Never)]` attribute on the class?